debug/src/debug.js

```diff
@@ -29,7 +29,17 @@ export function initDebug() {
 		let children = props && props.children;
 
 		if (type===undefined) {
-			throw new Error('Undefined component passed to createElement()\n'+serializeVNode(vnode));
+			throw new Error('Undefined component passed to createElement()\n\n'+
+			'You likely forgot to export your component or might have mixed up default and named imports'+
+			serializeVNode(vnode));
+		}
+		else if (type!=null && typeof type==='object') {
+			if (type._lastDomChild!==undefined && type._dom!==undefined) {
+				let info = 'Did you accidentilly passed a JSX Literal as JSX twice?';
```

Comment:
I think this should be `accidentally`

Reply:
I also think we would need an example here like <<App /> /> to be really verbose

Reply:
Something like `<<Foo />/>`?

Reply:
Yes, this explanation won't hit on the people who make this mistake which are often really new to jsx.

Reply:
```suggestion
				let info = 'Did you accidentally pass a JSX literal as JSX twice? (i.e. `let Bar = <Foo />; return <Bar/>`)';
```

Reply:
@ForsakenHarmony awesome, the idea to put the example in writing is really great! Changed the formatting a bit:

```
main.js:669 Uncaught Error: Invalid type passed to createElement(): [object Object]

Did you accidentally pass a JSX literal as JSX twice?

  let MyHome = <Home />;
  let vnode = <MyHome />;

This usually happens when you export a JSX literal and not the component.

[stack trace]
```